### PR TITLE
Citations and documentation improvements

### PR DESCRIPTION
**Pull request by erxleben**
Hi,

in the ccatwrapper is mentioned:
"If you use this tool in Galaxy, please cite Blankenberg D, et al. *In preparation.*"

Is it still in preparation?

Best
Anika

### CODE REVIEW

**Comment by jmchilton** [comment]
:+1:

**Comment by martenson** [comment]
Thanks Anika!

**Comment by bgruening** [comment]
Great! @blankenberg can you comment on the citation of this top secret manuscript? :)
"If you use this tool in Galaxy, please cite Blankenberg D, et al. In preparation."

**Comment by erasche** [comment]
Agreed, I want this citation so I can propagate it to https://github.com/galaxyproject/galaxy/pull/361

**Comment by nekrut** [comment]
This MS does not exist, so merge

File: tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml [hunk 1]
```diff
@@ -429,4 +429,9 @@ If you use this tool, please cite `Blankenberg D, Gordon A, Von Kuster G, Coraor
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml]: Here the old `**Citation**` has not been removed.

File: tools/velvet/velvetg.xml [hunk 2]
```diff
@@ -216,6 +216,7 @@
     </test>
   </tests>
   <help>
+**What it does**
 **Velvet Overview**
```
Comment (nsoranzo) [inline tools/velvet/velvetg.xml]: I'd remove `**Velvet Overview**`.

File: tools/velvet/velveth.xml [hunk 3]
```diff
@@ -73,6 +73,8 @@
     </test>
   </tests>
   <help>
+
+**What it does**
 **Velvet Overview**
```
Comment (nsoranzo) [inline tools/velvet/velveth.xml]: As above, I'd remove `**Velvet Overview**`.

**Comment by bgruening** [comment]
@nsoranzo ready to merge?

**Comment by nsoranzo** [comment]
Thanks for the fixes @bgruening !

**Pull request merged**